app/controllers/admin/oral_history_access_requests_controller.rb

```diff
@@ -0,0 +1,59 @@
+# STAFF FACING
+# Public facing actions are in app/controllers/oral_history_access_requests_controller.rb
+require 'csv'
+
+class Admin::OralHistoryAccessRequestsController < AdminController
+  def index
+    @oral_history_access_requests = Admin::OralHistoryAccessRequest.
+    where('created_at > ?', 3.months.ago).order(created_at: :desc).to_a
```

Comment:
Would it make more sense to show most recent N requests instead of 3 months? Say most recent 50 or something? `.order(created_at: :desc).limit(50)`?  Not sure.

Reply:
It occurred to me. Will ask Lee.

Reply:
No, past 3 months is fine, said Lee.

Reply:
I'm worried past 3 months would be TOO MANY lines, too big a page, performance problems. Since it's unbounded as to actual number, it just depends on how many requests there have been. Is why I would have made it a fixed number. But we can just deal with that if it becomes a problem, I guess.